utils/styles/globals.css

```diff
@@ -1,13 +1,32 @@
 html,
 body {
   margin: 0;
+  font-family: Roboto;
+  color: #121212;
+  background-color: #F5F5F5;
 }
 
 a {
   color: #0070f3;
   text-decoration: none;
 }
 
+a:hover {
+  color: #4BD2D2;
+  text-decoration: none;
+}
+
 * {
   box-sizing: border-box;
 }
+
+
+.css-12i7wg6-MuiPaper-root-MuiDrawer-paper {
+  background: linear-gradient(0deg, rgba(132,199,208,1) 0%, rgba(176,180,212,1) 50%, rgba(176,180,212,1) 100%);
+  color: #FFFFFF;
+  border: none;
+}
+
+.pageData {
+  margin-top: 100px;
```

Comment:
Double check that this margin is dynamic and works properly with that bar on the top. I'm sure it works but px are a hard unit of measurement so we have to be extra sure its stays responsive with changing page sizes and mobile screens.